tutorials/migrating/upgrading_to_godot_4.5.rst

```diff
@@ -231,6 +231,25 @@ Core
     the shortest arc between the two input vectors. Previously, it would return incorrect values for certain inputs
     (`GH-107618`_).
 
+Navigation
+~~~~~~~~~~
+
+.. note::
+
+    NavigationServer regions now by default update async using threads to improve performance.
+    This can cause additional delay in the update due to thread synchronisation.
+    The async region update can be toggled with the ``navigation/world/region_use_async_iterations`` project setting.
```

Comment:
```suggestion
    By default, the regions in a NavigationServer regions now update asynchronously using threads to improve performance.
    This can cause additional delay in the update due to thread synchronisation.
    The asynchronous region update can be toggled with the ``navigation/world/region_use_async_iterations`` project setting.
```
Is it one, or more than one update, too?